lib/index.js

```diff
@@ -4,13 +4,13 @@
  *
  * Configuration can be set by passing an object with openId and scim keys
  */
-module.exports = function(config = {})
-{
+module.exports = function (config = {}) {
   const AuthX = require('./connectors/openid')(config.openId, config.scim);
   config.openIdClient = AuthX;
 
   const services = require('./services')(config);
-  const routes = require('./routes')(config, AuthX, services);
+  const userEvents = require('./custom-events').userEvents;
+  const routes = require('./routes')(config, AuthX, services, userEvents);
 
-  return {AuthX: AuthX, services: services, routes: routes};
+  return { AuthX: AuthX, services: services, routes: routes, customEvents: events };
```

Comment:
```suggestion
  return { AuthX: AuthX, services: services, routes: routes, customEvents: userEvents };
```
The variable name `events` seems erroneous. Was this code tested?